actionmailer/test/parameterized_test.rb

```diff
@@ -55,15 +64,26 @@ class ParameterizedTest < ActiveSupport::TestCase
   end
 
   test "should enqueue a parameterized request with the correct delivery job" do
-    old_delivery_job = ParamsMailer.parameterized_delivery_job
```

Comment:
I think we should 🔥 this test, it doesn't make sense to have it anymore. I think the purpose of that test was to ensure that the `parameterized_delivery_job` was correctly respected but since you removed it ...

Reply:
You're right. I'll 🔥 it